src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -24,7 +24,7 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost
 {
     public class WebJobsScriptHostService : IHostedService, IScriptHostManager, IServiceProvider, IDisposable
     {
-        private readonly IApplicationLifetime _applicationLifetime;
+        private readonly AspNetCore.Hosting.IApplicationLifetime _applicationLifetime;
```

Comment:
Can we just alias this to avoid the namespaced type name throughout?

Reply:
Yes. I did that in a few spots. Not sure why I didn't do it here. I'll update it to be consistent.